gazebo_plugins/include/gazebo_plugins/gazebo_ros_imu.h

```diff
@@ -68,6 +68,9 @@ namespace gazebo
     /// \brief store link name
     private: std::string link_name_;
 
+    /// \brief store frame name
+    private: std::string frame_name_;
```

Comment:
:+1: Looks like all references to frame_id have been removed to fit upstream's parameter name.